hs_core/views/utils.py

```diff
@@ -1265,3 +1266,29 @@ def get_coverage_data_dict(source, coverage_type='spatial'):
             temporal_coverage_dict['start'] = start_date.strftime('%m-%d-%Y')
             temporal_coverage_dict['end'] = end_date.strftime('%m-%d-%Y')
         return temporal_coverage_dict
+
+
+def get_resource_metadata(resource):
```

Comment:
Is this so we don't have to do a full page refresh and adding files and setting a file type?